tutor/test_new_t1_14_TeacherReading.py

```diff
@@ -0,0 +1,2052 @@
+        closes_on = (today + datetime.timedelta(days=end)) \
+            .strftime('%m/%d/%Y')
+        assignment.assign_periods(
+            self.teacher.driver, {'all': (opens_on, closes_on)})
+
+
+        # add reading sections to the assignment
+        self.teacher.find(By.ID, 'reading-select').click()
+        self.teacher.wait.until(
+            expect.visibility_of_element_located(
+                (By.XPATH, '//div[contains(@class,"reading-plan")]')
+            )
+        )
+        assignment.select_sections(self.teacher.driver, ['1.1'])
+        self.teacher.find(
+            By.XPATH, '//button[text()="Add Readings"]'
+        ).click()
+
+        # publish
+        self.teacher.wait.until(
+            expect.visibility_of_element_located(
+                (By.XPATH, '//button[contains(@class,"-publish")]')
+            )
+        ).click()
+        try:
+            self.teacher.find(
+                By.XPATH,
+                '//label[contains(text(),"{0}")]'.format(assignment_name)
+            )
+        except NoSuchElementException:
+            self.teacher.find(
+                By.XPATH,
+                '//a[contains(@class,"header-control next")]'
+            ).click()
+            self.teacher.find(
+                By.XPATH,
+                '//label[contains(text(),"{0}")]'.format(assignment_name)
+            )
+
+        self.ps.test_updates['passed'] = True
+
+    # Case C132520 002 - Teacher | Save a draft for individual periods
+    @pytest.mark.skipif(str(132520) not in TESTS, reason='Excluded')
+    def test_teacher_save_a_draft_reading_for_individual_sections_132520(self):
+        """
+        Click on the 'Add Assignment' button
+        Click on the 'Add Reading' option
+        ***The teacher is taken to the Add Reading Assignment page*** (T1.14.001)
+
+        Enter an assignment name into the Assignment name text box
+        Click the 'Individual Sections' radio button
+        For each period reading is being assigned to:
+        Click on the Open Date text field and click on desired open date on calendar pop-up
+        Click on the Due Date text field and click on desired due date in the future on calendar pop-up
+        Click on the "+ Add Readings" button
+        Click on section(s) to add to assignment
+        Scroll to bottom of the page
+        Click on the "Add Readings" button
+        ***Takes user back to "Add reading" page *** (T1.14.004)
+        Click on the "Save As Draft" button (T1.14.005)
+
+        #Expected result
+        Takes user back to calendar dashboard. Assignment appears on user calendar dashboard on due date
+        with correct readings. 'draft' should appear before the assignment name.
+
+        """
+        # self.ps.test_updates['name'] = 't1.14.004' \
+        #                                + inspect.currentframe().f_code.co_name[4:]
+        # self.ps.test_updates['tags'] = ['t1', 't1.14', 't1.14.004', '7995']
```

Comment:
like the previous test case, update the name and tags as we need this in SauceLabs